sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionBasedLoadBalancer.java

```diff
@@ -286,10 +286,101 @@ private Mono<Void> loadBalance(final Tuple2<Map<String, PartitionOwnership>, Lis
                     return findPartitionToSteal(ownerPartitionMap);
                 });
 
-            claimOwnership(partitionOwnershipMap, ownerPartitionMap, partitionToClaim);
+            claimOwnership(partitionOwnershipMap, partitionToClaim);
         });
     }
 
+    /**
+     * This method make sure that checkpoint store works well and start processing messages of one partition.
+     */
+    Mono<Void> checkBlobStatus() {
```

Comment:
add test for this function?

Reply:
Not quite need to add the test, because there has been a client start hanlding checkpoint store error already.

Reply:
The partition based load balancer should only have methods related to load balancing... Is this better for CheckpointStore interface?  Additionally, these interfaces should contain methods that are agnostic of their implementation.  For example, if customers write an in-memory or Redis checkpoint store, they don't have a concept of "blob status".

Reply:
`checkBlobStatus` do load balance operation for one time to check everything works. If customers use their own checkpoint store, I think we also need to check their checkpoint store. Did I misunderstand?